packages/nuxt/src/app/composables/asyncData.ts

```diff
@@ -172,7 +180,7 @@ export function useAsyncData<
 
   asyncData.refresh = asyncData.execute = (opts = {}) => {
     if (nuxt._asyncDataPromises[key]) {
-      if (opts.dedupe === false) {
+      if (isDefer(opts.dedupe) || isDefer(options.dedupe)) {
```

Comment:
Let's allow overriding the global option.

```suggestion
      if (isDefer(opts.dedupe ?? options.dedupe)) {
```